src/routes/users/registerLogin.js

```diff
@@ -263,6 +263,10 @@ router.post(
         }
       });
 
+      if (!Object.keys(updateObj).length) {
+        res.status(400).send('INVALID_PAYLOAD');
+        return;
```

Comment:
I just realize we can actually
```suggestion
        throw new ValidationError('INVALID_PAYLOAD');
```
and let the error handler handle the http part
Sorry for missing this and please change to using ValidationError